src/trigger_settings.hpp

```diff
@@ -23,7 +23,10 @@
 #include "hardware_trigger.hpp"
 
 #include <QWidget>
+#include <QButtonGroup>
```

Comment:
Is `#include <QButtonGroup>` needed here?

Reply:
fixed